src/login/logind-button.c

```diff
@@ -77,10 +112,27 @@ int button_set_seat(Button *b, const char *sn) {
         return free_and_strdup(&b->seat, sn);
 }
 
-static void button_lid_switch_handle_action(Manager *manager, bool is_edge) {
+static ButtonModifierMask button_get_keycode_modifier_mask(int keycode)
+{
```

Comment:
coding style. we put the opening `{` on same line as function name, see all other code here as an example